src/misosoupy/import_sound_list.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Nov  2 10:53:23 2023
+
+@author: hhanse
+"""
+
+from __future__ import division
+
+import math
+import os  # handy system and path functions
+
+import numpy as np
+
+def function_import_sound_list(categorySpreadsheetName,numColumnsPerPage,numItemsPerColumn):
+    # Example:
+        # categorySpreadsheetName = "AllSoundCategories_v2.csv"
+        # numColumnsPerPage = 2 # How many columns will appear on the screen
+        # numItemsPerColumn = 10 # How many words (rows) will appear in each column
+
+    """
+    -------------------------------------------------------------------------------
+                            Set up paths
+    -------------------------------------------------------------------------------
+    """
+    
+    # Ensure that relative paths start from the same directory as this script
+    homeDir = os.path.dirname(os.path.abspath("__file__"))
+    os.chdir(homeDir)
```

Comment:
I would probably suggest have a function specifically for getting the path, and instead of changing the home directory, just save the absolute parent path and have that as an input to the other functions. Unless there is a specific reason you would want the home directory changed(?), this could have unintended consequences.